types/components/back-button/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import { FunctionalComponent } from 'vue';

```

Comment:
All type files should be removed from github, there's no point in having them here